packages/datadog_tracking_http_client/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## Unreleased
+## 1.0.1-beta.1
```

Comment:
I'm curious why not `1.0.0-beta.2`. Does it mean that from now on we will continue only with `1.0.1`?

Reply:
Because there's a possibility people will want / need to stay on Flutter 2.10. If I use beta.2 I can't continue support for the pre-Flutter 3.0 version line.

Until Flutter 2.10 is not being used, I'll likely have to maintain a 1.0.0 and a 1.0.1 release line. (TBH, it probably should have been 1.1.0, but that ship has sailed)

Reply:
Thanks, it is clear now!